kubernetes/doc/rbac.md

```diff
@@ -0,0 +1,73 @@
+
+IoT Edge runtime on Kubernetes leverages standard [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) method to regulate access to resources inside Kubernetes cluster. Edge runtime installs itself in a namespace provided by user. All resources it creates during installation and work are scoped to the namespace.
+
+```
++----------------+       +-------------------------------+      +--------------------------------------+
+| ServiceAccount |---+---| ClusterRoleBinding            |------| ClusterRole                          |
+| iotedged       |   |   | iotedge:{name}:auth-delegator |      | system:auth-delegator                |
++----------------+   |   +-------------------------------+      +--------------------------------------+
+                     |   +-------------------------------+      +--------------------------------------+
+                     +---| ClusterRoleBinding            |------| ClusterRole                          |
+                     |   | iotedge:{name}:node-observer  |      | iotedge:{name}:iotedge:node-observer |
+                     |   +-------------------------------+      +--------------------------------------+
+                     |   +-------------------------------+      +--------------------------------------+
+                     +---| RoleBinding                   |------| Role                                 |
+                         | iotedged                      |      | iotedged                             |
+                         +-------------------------------+      +--------------------------------------+
+
+
++----------------+       +-------------------------------+      +--------------------------------------+
+| ServiceAccount |-------| RoleBinding                   |------| Role                                 |
+| edgeagent      |       | edgeagent                     |      | edgeagent                            |
++----------------+       +-------------------------------+      +--------------------------------------+
+```
+
+## iotedged
+
+An iotedged component requires the most privileged entity in the Edge deployment so it has ClusterRole but with very limited scope of roles. The amount of permissions is related to the amount of operations it needs to successfully install EdgeAgent and monitor its running status. The full list of permissions required for iotedged can be found in the [main repository](../charts/edge-kubernetes/templates/edge-rbac.yaml). 
+
+In addition to standard permissions to list, create, delete, update and watch Kubernetes resources like Deployments, Pods, Services, ConfigMaps etc it has security related permissions.
+
+An iotedged has a ClusterRole because it is required to perform cluster wide operations. When reporting system information, it lists nodes and collects all unique types of architecture with number of nodes of this type. Another cluster-wide operation is TokenReview that is a foundation of module authentication.
```

Comment:
```suggestion
`iotedged` has a ClusterRole because it is required to perform the following cluster wide operations:
* When reporting system information, it lists nodes and collects all unique types of architecture with number of nodes of this type. 
* TokenReview that is the foundation of module authentication.
```